src/pages/dashboard/_util.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/prefer-default-export */
+import { format } from 'd3-format'
+import { timeFormat, timeParse } from 'd3-time-format'
+
+export const formatDate = timeFormat('%b. %e')
+export const formatNumber = format(',')
+export const parseDate = timeParse('%Y%m%d')
+export const calculateTotal = d => {
+  return d.positive + (d.negative || 0)
+}
+export const getStateName = abbr => {
+  const names = {
```

Comment:
The two letter state abbreviation to state name is contained within the data returned by this endpoint: https://covidtracking.com/api/v1/states/info.json

I'd say it's better to use that than have a hrardcoded mapping.  Perhaps this is something we can remove after converting the code to use the gatsby data.

Reply:
I'm down! Though I'd love to figure out how to do the GraphQL query so it happens at build time and doesn't require another request from the browser